src/toolkit/matl_buy_policy.h

```diff
@@ -78,12 +78,23 @@ class MatlBuyPolicy : public Trader {
   /// be sent to fill the buffer's empty space.
   /// @warning, (s, S) policy values are ambiguous for buffers with a capacity
   /// in (0, 1]. However that is a rare case.
+  /// The following features, if used, will set the policy to so that agents
+  /// cycle through "on" phases, where the buf will attemp to be filled, and
+  /// "off" or dormant phases, where no requests will be made regardless of
+  /// capacity.  The "on" and "off" phases are sampled and rounded to the
+  /// nearest integer number of time steps from a truncated normal
+  /// distribution from a mean, standard deviation, min, and max value.
+  /// @param active the length of the on, actively buying period
+  /// @param dormant the length of the dormant period
   /// @{
   MatlBuyPolicy& Init(Agent* manager, ResBuf<Material>* buf, std::string name);
   MatlBuyPolicy& Init(Agent* manager, ResBuf<Material>* buf, std::string name,
                       double throughput);
   MatlBuyPolicy& Init(Agent* manager, ResBuf<Material>* buf, std::string name,
                       double fill_to, double req_when_under);
+  MatlBuyPolicy& Init(Agent* manager, ResBuf<Material>* buf, std::string name,
```

Comment:
If there are default values for `active` and `dormant` that preserve the behavior of not having them at all (e.g. `active = 1` and `dormant = 0`?) then perhaps we can just add these arguments to a previous constructor with defaults?